test/functional/plugin/shada_spec.lua

```diff
@@ -715,6 +722,7 @@ describe('In autoload/shada.vim', function()
         '  # Unexpected enum value: expected one of 0 (CHARACTERWISE), '
         .. '1 (LINEWISE), 2 (BLOCKWISE)',
         '  + rt   type         10',
+        '  + ru   is_unnamed   FALSE',
```

Comment:
What exactly do I need to add for these tests? Is it that I need to test that it errors correctly for each of the non-boolean messagepack types? I only had one with an incorrect type (passed an int instead of a boolean), so should I add more?

I can't find the  case where the values of each key are the default. Can you post the line number?